Sources/App/Controllers/User/PushNotificationController.swift

```diff
@@ -0,0 +1,53 @@
+import Vapor
+import FluentProvider
+import Foundation
+import Multipart
+
+final class PushNotificationController {
+
+  func subscribe(_ request: Request) throws -> ResponseRepresentable {
+
+    guard
+      let json = request.json,
+      let pushToken = json[Client.Keys.pushToken]?.string
+    else {
+      throw Abort(.badRequest, reason: "Request's body no have pushToken to the push-notification registration")
+    }
+
+    if let client = try Client.returnIfExcists(request: request) {
```

Comment:
typo in `Client.returnIfExcists`.  need - `Client.returnIfExists`

Reply:
fixed, thank you!